python_modules/airline-demo/airline_demo/resources.py

```diff
@@ -214,3 +214,10 @@ def _do_load(data_frame, table_name):
         dialect='postgres',
         load_table=_do_load,
     )
+
+
+if __name__ == '__main__':
+    # This is a brutal hack. When the SparkSession is created for the first time there is a lengthy
+    # download process from Maven. This allows us to run python -m airline_demo.resources in the
+    # Dockerfile and avoid a long runtime delay before each containerized solid executes.
+    spark_session_local.resource_fn(None)
```

Comment:
saw you're also doing this in `test_types.py`:

`spark = _spark_context()['test'].resources['spark'].resource_fn(None)`

since `_spark_context()` uses `spark_session_local` won't the above break the tests?

Reply:
no -- maybe we should talk through what this is doing -- it's just initializing the resource (which is idempotent, but incurs the download-from-maven cost the first time it's called)